src/analysis/retail/hunter/shared/talents/ExplosiveShot.tsx

```diff
@@ -63,6 +65,25 @@ class ExplosiveShot extends Analyzer {
       </Statistic>
     );
   }
+  get guideSubsectionSV() {
+    const explanation = (
+      <p>
+        <strong>
+          <SpellLink spell={TALENTS.EXPLOSIVE_SHOT_TALENT} />
+        </strong>{' '}
+        deals damage after 3 second delay <SpellLink spell={TALENTS.TIP_OF_THE_SPEAR_TALENT} />.
```

Comment:
```suggestion
        deals damage after 3 second delay.
```

this looks like an accidental paste?